fontra2fontir/src/toir.rs

```diff
@@ -22,7 +22,14 @@ pub(crate) fn to_ir_static_metadata(
     let axes = font_data
         .axes
         .iter()
+        .map(|a| match a {
+            crate::fontra::FontraAxis::Discrete(_) => Err(WorkError::UnsupportedConstruct(
```

Comment:
Suggest adding a minimal example of a fontra font with discrete axes to testdata and a test to prove we reject it.